dnn/variational_inference.ipynb

```diff
@@ -3,31 +3,47 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        unscaled_mean = mean_output * mean_scaler.var_ + mean_scaler.mean_
</pre><p>If we're predicting residuals, this should be computed over the residuals instead of the raw values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/165/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>